spring-cql/src/test/java/org/springframework/cassandra/test/integration/config/java/CqlTemplateConfigIntegrationTest.java

```diff
@@ -32,44 +31,38 @@
 /**
  * @author Matthews T. Adams
  * @author Oliver Gierke
+ * @author Mark Paluch
  */
-public class CqlTemplateConfigIntegrationTest extends AbstractKeyspaceCreatingIntegrationTest {
-
-	public static final String KEYSPACE_NAME = Utils.randomKeyspaceName();
+public class CqlTemplateConfigIntegrationTest extends AbstractEmbeddedCassandraIntegrationTest {
 
 	@Configuration
 	public static class Config extends AbstractCqlTemplateConfiguration {
 
 		@Override
 		protected String getKeyspaceName() {
-			return KEYSPACE_NAME;
+			return "system";
 		}
 
 		@Override
 		protected int getPort() {
-			return CASSANDRA_NATIVE_PORT;
+			return PROPS.getCassandraPort();
 		}
 	}
-	
+
 	Session session;
 	ConfigurableApplicationContext context;
-	
+
 	@Before
 	public void setUp() {
-		
 		this.context = new AnnotationConfigApplicationContext(Config.class);
 		this.session = context.getBean(Session.class);
 	}
-	
+
 	@After
 	public void tearDown() {
 		context.close();
 	}
 
-	public CqlTemplateConfigIntegrationTest() {
-		super(KEYSPACE_NAME);
-	}
-
 	@Test
 	public void test() {
 		IntegrationTestUtils.assertCqlTemplate(context.getBean(CqlTemplate.class));
```

Comment:
The assertion is not necessary since the `ApplicationContext.getBean(:Class);` method will throw a `BeansException` if the bean with type is not found.

http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/beans/factory/BeanFactory.html#getBean-java.lang.Class-